rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: Tuple[str, str, str],
+            contract: EthereumContract,
+    ) -> List:
```

Comment:
I believe we avoid having just List returned as type. Should be `List[Optional[Union[str, int]]]` I believe